math/src/main/java/com/powsybl/math/timeseries/TimeSeriesConstants.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.math.timeseries;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public final class TimeSeriesConstants {
+
+    private TimeSeriesConstants() {
+    }
+
+    public static final char DEFAULT_SEPARATOR = ';';
```

Comment:
This constant should probably be defined before the private constructor?

Reply:
Done